ansible/wsu.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- hosts: localhost
+  vars:
+    openshift_node_machineconfigpool: 'worker'
+    openshift_node_bootstrap_port: 22623
+    openshift_node_bootstrap_endpoint: "{{ openshift_node_bootstrap_server }}:{{ openshift_node_bootstrap_port }}/config/{{ openshift_node_machineconfigpool }}"
+    
+  tasks:
+    - name: Check parameters
+      fail:
+          msg: "Please provide all the following parameters to run this playbook [ kubelet_location, kubeovn_location, wmcb_location, openshift_node_bootstrap_server ]"
+      when: kubelet_location is not defined or
+             kubeovn_location is not defined or
+             wmcb_location is not defined or
+             openshift_node_bootstrap_server is no defined
+ 
+    - name: Show ignition file endpoint 
+      debug:
+        msg: "{{ openshift_node_bootstrap_endpoint }}"
+
+    - name: Wait for bootstrap endpoint to show up
+      uri:
+        url: "{{ openshift_node_bootstrap_endpoint }}" 
+        validate_certs: false
+      delay: 10
+      retries: 60
+      register: result
+      until:
+        - result.status is defined
+        - result.status == 200
+   
+    - name: Fetch bootstrap ignition file locally
+      uri:
+        url: "{{ openshift_node_bootstrap_endpoint }}"
+        dest: /home/bootstrap.ign
+        validate_certs: false
+      register: bootstrap_ignition
+
+    - name: Download Kubelet to controller machine
+      get_url:
+        url: {{ kubelet_location }} 
+        dest: /home/kubelet.exe
```

Comment:
You cannot use `/home` as a destination. Create a temp location using the [tempfile](https://docs.ansible.com/ansible/latest/modules/tempfile_module.html) module.